src/vs/platform/extensionManagement/common/extensionTipsService.ts

```diff
@@ -148,6 +148,10 @@ export abstract class AbstractNativeExtensionTipsService extends ExtensionTipsSe
 					this.allOtherExecutableTips.set(key, { exeFriendlyName: exeBasedExtensionTip.friendlyName, windowsPath: exeBasedExtensionTip.windowsPath, recommendations: otherRecommendations });
 				}
 			});
+			console.log('these recommendations(both highImportanceRecommendations and  mediumImportanceRecommendations) always seem to be empty: ');
```

Comment:
I wish I can get these to show up in my dev tools!
I've set my log level to trace, did fresh build, but still cannot see any logs nor get the breakpoints to hit here.

Perhaps it has to do something with being abstract class constructor? I'm pretty lost.

Reply:
Use ILogService instead and you can see them in the shared log view